lib/daemon/index.js

```diff
@@ -98,10 +132,27 @@ function destroyFilePid(filename) {
 	}
 }
 
+function getCommandAppStatus() {
+	return !!getFilePid(CMD_LOCK_FILE);
+}
+
 
 function App(runstate, pid) {
+	// Force color output on logs - this is necessary so
+	// that the daemonized master process as well as its workers
+	// output colorized logs when running start/stop/restart/etc.
+	process.env.FORCE_COLOR = 1;
```

Comment:
This probably has some drawbacks, no?

Reply:
You are right. What we can do is have the daemonizer filter colors if it doesn't support them.

Reply:
Just added the code to do that to this PR